Assets/MixedRealityToolkit/Interfaces/SceneSystem/IMixedRealitySceneSystemEditor.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.SceneSystem;
+
+/// <summary>
+/// Optional editor-only interface for use with facade inspectors.
+/// If a scene system service does not implement this interface, the facade will not be rendered.
+/// </summary>
+public interface IMixedRealitySceneSystemEditor
+{
+#if UNITY_EDITOR
+    /// <summary>
+    /// Returns all content scene tags in the scene service profile.
+    /// </summary>
+    IEnumerable<string> ContentTags { get; }
+
+    /// <summary>
+    /// Returns the content scenes in the scene service profile.
+    /// </summary>
+    SceneInfo[] ContentScenes { get; }
+
+    /// <summary>
+    ///  Returns the lighting scenes in the scene service profile.
+    /// </summary>
+    SceneInfo[] LightingScenes { get; }
+    
+    /// <summary>
+    /// Loads the next content scene in-editor. Use instead of LoadNextContent while not in play mode.
+    /// </summary>
+    /// <param name="wrap"></param>
```

Comment:
>/// <param name="wrap"></param> [](start = 4, length = 31)

Can you add a comment about what wrap does? i.e. I assume this means it will wrap around back to the beginning when already at the end, but it could also mean "wrap this scene in some other scene?" or something else non-sensical and we may as well get rid of any room for doubt